apps/api/src/app/benchmark/benchmark.service.ts

```diff
@@ -232,7 +252,7 @@ export class BenchmarkService {
           dataSource,
           symbol,
           date: {
-            gte: startDate
+            in: dates
```

Comment:
This does not seem to work. `marketDataItems` is `[]` and thus this error message in the log: `No historical market data has been found for`...